modules/default/compliments/compliments.js

```diff
@@ -63,20 +63,28 @@ Module.register("compliments", {
 				self.config.compliments = JSON.parse(response);
 				self.updateDom();
 			});
-		} else if (this.config.advice) {
-			var xobj = new XMLHttpRequest();
-			xobj.overrideMimeType("application/json");
-			xobj.open("GET", "https://api.adviceslip.com/advice", true);
-			xobj.onreadystatechange = function() {
-				if (xobj.readyState === 4 && xobj.status === 200) {
-					const adviceResp = JSON.parse(xobj.responseText);
-					self.config.compliments = adviceResp.slip.advice
-					self.updateDom();
-				}
-			};
-			xobj.send(null);
 		}
 
+    if (this.config.advice) {
+        var xobj = new XMLHttpRequest();
+        xobj.overrideMimeType("application/json");
+        xobj.open("GET", "https://api.adviceslip.com/advice/search/a", true);
+        xobj.onreadystatechange = function() {
+        if (xobj.readyState === 4 && xobj.status === 200) {
+            const adviceResp = JSON.parse(xobj.responseText);
+            const anytime = []
+            adviceResp.slips.forEach(item => {
+                if (item.advice.length < 30) {
```

Comment:
maybe make the number of advices configurable?

Reply:
This is not the number of advices, rather the length of the string so that we dont get pieces of advice running over 5 lines in huge font in the middle of the screen, for example.

Reply:
Ah, I see. Adding a little comment to that line with that info would be useful for later generations I guess :-)